### ISSUE TITLE
Windows wheel build fail due to chocolatey failure.

### ISSUE THREAD

**Issue by charris** [numpy/numpy#25980]
```
Chocolatey v2.2.2
Installing the following packages:
pkgconfiglite
By installing, you accept licenses for the packages.
Invalid credentials specified.
Please provide credentials for: https://community.chocolatey.org/api/v2/
```
This might not be the essential failure. See https://github.com/numpy/numpy/actions/runs/8217793492/job/22473896158

**Comment by charris** [comment]
This may be due to #25861, but I expect it would have shown up earlier.

**Comment by charris** [comment]
The manual builds on main seem to work, so I suspect that the problem is running on a  maintenance branch.

**Comment by andyfaff** [comment]
Definitely an upstream change, there's nothing we've done to cause this.

**Comment by charris** [comment]
Might be some service/network failure.

**Comment by andyfaff** [comment]
Yeah, I suggest we revisit tomorrow and see if things have reset.

**Comment by mattip** [comment]
I tried restarting one of the jobs, it still fails. The [image being used](https://github.com/numpy/numpy/actions/runs/8219045556/job/22480712991#step:1:9) was updated March 4, maybe connected? Although the update to chocolatey 2.2.2 was done [6 months ago](https://github.com/actions/runner-images/pull/8095), so maybe not.

**Comment by andyfaff** [comment]
I think we would've seen issues earlier if it was to do with the image. Besides there are azure tests failing, I think that means it's the chocolatey server?

**Comment by andyfaff** [comment]
There are some Windows jobs (that use pkgconfiglite) that are succeeding, but some that are failing. Both of them appear to use the same image:

>   Image: windows-2019
  Version: 20240304.2.0

**Comment by rgommers** [comment]
Hmm, not the first time. It usually resolves itself, but it's a bit frustrating. Not much visibility into the current status of Chocolatey, no issue tracker - https://chocolatey.org/support#foss lists options, but the mailing list is closed and Discord requires signing up too. 

The issue is https://github.com/actions/runner-images/issues/9476. It was reported as fixed there 7 hours ago (2 hours after the last comment above. I restarted the failing builds that @charris linked to in the issue description, and they all passed the `choco install` stage: https://github.com/numpy/numpy/actions/runs/8217793492.

So we should be good here, closing.

That said, I'd like to get rid of Chocolatey as much as possible. I have some WIP work to build `pkgconf` wheels, which would take care of most of it - the only usage that is harder to get rid of is the Clang-cl install in one Windows CI job.

**Comment by mattip** [comment]
> I have some WIP work to build pkgconf wheels

Sounds good. Could we use conda to get clang-cl/gfortran instead of choco? I see conda-forge has [pkg-config](https://anaconda.org/conda-forge/pkg-config)

**Comment by rgommers** [comment]
> Could we use conda to get clang-cl/gfortran instead of choco? I see conda-forge has [pkg-config](https://anaconda.org/conda-forge/pkg-config)

Yes, good point. For `pkg-config` alone it doesn't quite make sense to use conda-forge, because it'd turn a 500 kb download into a ~50 MB one. But for Clang-cl it makes sense - and we're actually using Clang-cl in conda-forge to build SciPy on Windows.

**Comment by rgommers** [comment]
> Using an external package manager (conda/choco) makes more sense to me than making a pkgconf wheel, as it's one more thing we have to look after.

It's about as simple as it gets in building binaries for non-Python programs. I tried it because when setting up a new Windows machine I gave myself the challenge of trying it without conda channels. And the experience is pretty terrible. Using conda-forge is the only reasonable dev experience on Windows right now. http://scipy.github.io/devdocs/building/index.html#system-level-dependencies isn't complete enough, it doesn't give you BLAS/LAPACK nor pkg-config. So if we make `scipy-openblas32` wheels work, all we need there still is `pkg-config`.
